lib/semantics/check-omp-structure.cc

```diff
@@ -807,8 +826,14 @@ void OmpStructureChecker::Enter(const parser::OmpAlignedClause &x) {
 void OmpStructureChecker::Enter(const parser::OmpDefaultClause &) {
   CheckAllowed(OmpClause::DEFAULT);
 }
-void OmpStructureChecker::Enter(const parser::OmpDefaultmapClause &) {
+void OmpStructureChecker::Enter(const parser::OmpDefaultmapClause &x) {
   CheckAllowed(OmpClause::DEFAULTMAP);
+  using VariableCategory = parser::OmpDefaultmapClause::VariableCategory;
+  if (!std::get<std::optional<VariableCategory>>(x.t)) {
+    context_.Say(GetContext().clauseSource,
+        "The scalar VARIABLECATEGORY must be specified on the DEFAULTMAP "
```

Comment:
Users may not be able to understand this, how about `The argument TOFROM:SCALAR must be specified on the DEFAULTMAP clause` ?

Reply:
Yes, that sounds better. I did think as I wrote it that it didn't read that well but I couldn't decide what to change it to